src/test/java/SeleniumTests.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.Assert;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SeleniumTests {
+
+
+    private static String chromeDriverPath = "C:\\chromedriver_win32\\chromedriver.exe";
```

Comment:
never use the absolute path to a file in a project. Use only the path from content root. Your path doesn't work for me. BTW please add follow dependency
```
<dependency>
            <groupId>io.github.bonigarcia</groupId>
            <artifactId>webdrivermanager</artifactId>
            <version>5.1.1</version>
        </dependency>
``` 
than you can use this code 
```
WebDriverManager.getInstance(ChromeDriver.class).setup();
  driver = new ChromeDriver();
```
this will help you avoid having to specify the path to the driver and monitor its updates. So that you can remove `chromeDriverPath` and `setChromeDriverPath()` method